### ISSUE TITLE
49856347: [UIAccessibilityTraits] Allow use of the Accessibility Trait to define custom containers for the "containers" rotor option

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#21183]
#### Description

Summary:
Allow developers to improve VoiceOver navigation by permitting us to expose views to the "containers" rotor option.
In other words, expose the 0x200000000000 Accessibility Trait that you are using internally to define views (such as UINavigationBar, UITabBar, UITableView, etc.) 

Expected Results:
A developer should be able to define views with an Accessibility Trait that exposes them to be navigable using the "containers" rotor option.

Actual Results:
Developers cannot define a view as a container for the "containers" rotor option. Therefore, certain views will not be readily accessible to VoiceOver users navigating the application using this feature.

Important Note:
Such Accessibility Trait exists because it is used by numerous system views, but is not freely available to developers. Nevertheless, using the private 0x200000000000 Accessibility Trait may have our applications rejected for using private behaviors of the SDK. Or even worse, since it is not public, the implementation may change and break the application functionality at a later time. 
However, please notice that we could copy the traits of a system view (let's say, a UITabBar) that has that trait, and we would not be using any private interface to obtain the private behavior.

Version/Build:
Tested over iOS 11 and 12.

Thanks so much for your time!!!

-
Product Version: iOS 11/12
Created: 2019-04-12T17:56:17.265373
Originated: 2019-04-12T00:00:00
Open Radar Link: http://www.openradar.me/49856347

**Comment by openradar-mirror** [comment]
Modified: 2019-04-23T23:13:30.955176

**Comment by openradar-mirror** [comment]
Modified: 2019-04-23T23:13:30.955176

**Comment by openradar-mirror** [comment]
Modified: 2019-04-23T23:13:30.955176

**Comment by openradar-mirror** [comment]
Modified: 2019-04-23T23:13:30.955176

**Comment by openradar-mirror** [comment]
Modified: 2019-04-23T23:13:30.955176

**Comment by openradar-mirror** [comment]
Modified: 2019-04-23T23:13:30.955176

**Comment by openradar-mirror** [comment]
Modified: 2019-04-23T23:13:30.955176

**Comment by openradar-mirror** [comment]
Modified: 2019-04-23T23:13:30.955176

**Comment by openradar-mirror** [comment]
Modified: 2019-04-23T23:13:30.955176

**Comment by openradar-mirror** [comment]
Modified: 2019-04-23T23:13:30.955176
